share/completions/climate.fish

```diff
@@ -0,0 +1,81 @@
+# fish completion for climate
+# climate is a tool that provide simple commands that wrap some useful
+# but complex combination of Linux commands.
+# The tool can be found at https://github.com/adtac/climate
+
+function __fish_is_git_repository
```

Comment:
It's already implemented in [git.fish](https://github.com/fish-shell/fish-shell/blob/master/share/completions/git.fish#L38). Maybe split it?